Grid/Group/GroupColumn.php

```diff
@@ -29,9 +29,12 @@ public function setColumnClassCmb2() {
 			//\Cmb2Grid\Cmb2\Utils::initializeFieldArg( $field->args['fields'][$fieldID], 'before_row' );
 			//\Cmb2Grid\Cmb2\Utils::initializeFieldArg( $field->args['fields'][$fieldID], 'after_row' );
 
-
-			@$field->args['fields'][ $fieldID ]['before_row'] .= "<div class=\"{$columnClass}\">";
-			@$field->args['fields'][ $fieldID ]['after_row']  .= '</div>';
+			if ( !empty( $fieldID['before_row'] ) ) {
```

Comment:
Better to combine both checks into one `if` statement as having the closing html tag without the opening one or visa versa will create broken html.